### PR TITLE
[minor] update everything

### PR DESCRIPTION
**Pull request by SimonSchick**
Mostly linting changes, primarily problem is the error inheritance that I had to fix up...

### CODE REVIEW

File: src/errors/ReadError.ts [hunk 1]
```diff
@@ -6,8 +6,14 @@ import { BormError } from './BormError';
  * or malformed value was received instead.
  */
 export class ReadError extends BormError {
-    constructor (message: string, public readonly model: IModel, public readonly property: string, public readonly value: any) {
+    constructor (
+        message: string,
+        public readonly model: IModel,
+        public readonly property: string,
```
Comment (saadataz) [inline src/errors/ReadError.ts]: are we using this anywhere?

**Comment by saadataz** [comment]
Looks good otherwise

File: src/errors/ReadError.ts (hunk 1 above)
Comment (SimonSchick) [inline src/errors/ReadError.ts]: No, but it's metadata for the error.

Comment (saadataz) [inline src/errors/ReadError.ts]: oh ok. got it

**Pull request merged**